modules/btcpayserver.nix

```
@@ -84,6 +84,12 @@ in {
         default = null;
         description = "The lightning node implementation to use.";
       };
+      rootpath = mkOption {
+        type = types.nullOr types.str;
+        default = null;
+        example = "/btcpayserver";
+        description = "The root path in the URL to access BTCPay.";
```

```suggestion
        description = "The root path in the URL to access btcpayserver.";
```
We're using `btcpayserver` instead of `BTCPay` in the other descriptions.

---

Fixed in a0446c8c11608b620391e819fb19f703a1ee7b13